api/core/workflow/nodes/iteration/entities.py

```diff
@@ -1,10 +1,17 @@
+from enum import Enum
 from typing import Any, Optional
 
 from pydantic import Field
 
 from core.workflow.nodes.base import BaseIterationNodeData, BaseIterationState, BaseNodeData
 
 
+class ErrorHandleMode(str, Enum):
+    TERMINATED = "Terminated"
+    CONTINUE_ON_ERROR = "Continue on error"
+    REMOVE_ABNORMAL_OUTPUT = "Remove abnormal output"
```

Comment:
The enumeration here can use lowercase letters + hyphen (-), just like in other places.